tools/tm-signer-harness/README.md

```diff
@@ -0,0 +1,137 @@
+# tm-signer-harness
```

Comment:
^ ++ or another method

I also just noticed that we have duplicate files; e.g., `/tools/tm-bench/README.md` is also `/docs/tools/benchmarking.md` and same goes for `tm-monitor`.

Is there a better way we can do this? Ideally the files are in `/docs` so that they can easily be included in the build. Then the various README's within `/tools` can link to those files?

see also #1921